doc/architecture/vulnerability-management.rst

```diff
@@ -0,0 +1,146 @@
+Ostro OS vulnerability handling
```

Comment:
add this as first line of the file (this will make inter-document linking to this document possible):

.. _vulnerability-management: